src/ck-core/ck.C

```diff
@@ -3043,11 +3065,11 @@ void CkMessageWatcherInit(char **argv,CkCoreState *ck) {
 
 int CkMessageToEpIdx(void *msg) {
```

Comment:
It'll be good to fix the weird indentation within this function since you're already touching this code.